conjure-java-undertow-runtime/src/main/java/com/palantir/conjure/java/undertow/runtime/ConjureContexts.java

```diff
@@ -60,6 +60,13 @@ private static final class ConjureServerRequestContext implements RequestContext
             this.requestArgHandler = requestArgHandler;
         }
 
+        @Override
+        public String requestTarget() {
+            String requestUri = exchange.getRequestURI();
+            String queryString = exchange.getQueryString();
+            return queryString.isEmpty() ? requestUri : requestUri + "?" + queryString;
```

Comment:
Slight preference for the more defensive `Strings.isNullOrEmpty(` in case of future changes. We have similar (seemingly impossible) checks in our logging layouts where such cases are often triggered by incomplete mockito stubs.

```suggestion`
            return Strings.isNullOrEmpty(queryString) ? requestUri : requestUri + "?" + queryString;
```